config/routes.rb

```diff
@@ -43,4 +40,20 @@
       resources :group_assignments, path: 'group-assignments'
     end
   end
+
+  namespace :stafftools do
+    mount Sidekiq::Web  => '/sidekiq'
```

Comment:
@mastahyeti Out of curiosity, what's your reason for preferring controller level auth rather than route constraints?